doc/en/exploitation_guide/04realtime.rst

```diff
@@ -337,7 +337,7 @@ The table below gives a description of all the columns of the table displayed wh
 +--------------------+-------------------------------------------------------------------------------------------------------------------------+
 | Host               | Displays the name of host. The |url_link| icon allows to access to host's page details.                                 |
 +--------------------+-------------------------------------------------------------------------------------------------------------------------+
-| Services           | Displays the name of service. The |nonotifications| icon indicates that notifications are disabled for this service.    |
+| Services           | Displays the name of service. The |notifications| icon indicates that notifications are disabled for this service.    |
```

Comment:
@thiuyendang  I need review this, because these is a reference to a image.

Reply:
Hello Luiz, effectively, the table images for hosts and services including references to icons are not displayed anymore in your version of documentation.